schema/crdb/dbinit.sql

```diff
@@ -1107,6 +1107,15 @@ CREATE TABLE IF NOT EXISTS omicron.public.instance (
      */
     auto_restart_policy omicron.public.instance_auto_restart,
 
+    /*
+     * Which disk, if any, is the one this instance should be directed to boot
+     * from. With a boot device selected, guest OSes cannot configure their
+     * boot policy for future boots, so also permit NULL to indicate a guest
+     * does not want our policy, and instead should be permitted control over
+     * its boot-time fates.
+     */
+    boot_device STRING(63),
```

Comment:
`omicron/schema/crdb/README.adoc` is good! this draft PR just predates me reading it and understanding where we put migrations. so `dbinit.sql` is updated but there's no new `KnownVersion` and `schema` tests correctly fail over it.

Reply:
(it's a UUID now. and i've got the rest of schema management sorted out i think.)